src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/unit/restapi/controller/ReferralControllerTest.kt

```diff
@@ -458,7 +495,7 @@ constructor(
     }.andReturn()
 
     val jsonResponse = mvcResult.response.contentAsString
-    val paginatedReferralSummary = jacksonObjectMapper()
+    val paginatedReferralSummary = objectMapper
```

Comment:
Always use the Kotlin-implementation `jacksonObjectMapper()` where possible, as it's implemented as a singleton and runs much more efficiently than manually rolling it.